cloudnetwork/clientset/versioned/scheme/register.go

```diff
@@ -0,0 +1,40 @@
+// Code generated by client-gen. DO NOT EDIT.
+
+package scheme
+
+import (
+	cloudv1 "github.com/openshift/api/cloudnetwork/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	schema "k8s.io/apimachinery/pkg/runtime/schema"
+	serializer "k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+var Scheme = runtime.NewScheme()
+var Codecs = serializer.NewCodecFactory(Scheme)
+var ParameterCodec = runtime.NewParameterCodec(Scheme)
+var localSchemeBuilder = runtime.SchemeBuilder{
+	cloudv1.AddToScheme,
```

Comment:
This is a problem as no such variable has been defined here: https://github.com/openshift/api/blob/master/cloudnetwork/v1/register.go#L9-L14

Reply:
This has now been fixed by https://github.com/openshift/api/pull/890